packages/graphql/src/schema-model/annotation/Annotation.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { CypherAnnotation } from "./CypherAnnotation";
+
+export type Annotation = CypherAnnotation;
```

Comment:
This same approach is taken in the cypher builder for a reason.

Interfaces in typescript are really limited, something that is not an interface can match that interface. Currently, we don't have a common interface for annotations. Meaning that the class structure would look like this:

```ts
interface Annotation {}

class CypherAnnotation implements Annotation {}
```

The problem with this is that then:
```ts
function myFunc(an: Annotation) {}

myFunc({}); // This is ok, because an empty object (and any object) matches the interface Annotation
myFunc(""); // This is also ok
```

By setting a type union of the classes we want to match, we ensure that only those will be added. In the future, as we refine the common structure, we will may move to a more classy structure (if needed).